src/server/handler/request.rs

```diff
@@ -583,18 +583,19 @@ impl Server {
             _ => return,
         };
 
-        let internal_err = |err: String| {
+        let internal_err = |message: String| {
             self.respond(Response {
                 id: id.clone(),
                 result: None,
                 error: Some(ResponseError {
                     code: -32603,
-                    message: err,
+                    message,
                     data: None,
                 }),
             });
         };
 
+        let is_clang_format = self.args.fmt_exe.ends_with("clang-format");
         let mut code = String::new();
```

Comment:
It's better use 'contains' rather than 'ends_with'. May be contain '.exe' extension on Windows.